crates/stackable-operator/src/product_logging/framework.rs

```diff
@@ -589,6 +589,7 @@ pub fn create_logback_config(
     </rollingPolicy>
     <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
       <MaxFileSize>{max_log_file_size_in_mib}MB</MaxFileSize>
+      <checkIncrement>5s</checkIncrement>
```

Comment:
`5s` sets `checkIncrement` to 5 milliseconds.

The plural `s` is ignored in the pattern of the unit part:
https://github.com/qos-ch/logback/blob/v_1.5.7/logback-core/src/main/java/ch/qos/logback/core/util/Duration.java#L43

Therefore, this value has effectively no unit which defaults to milliseconds:
https://github.com/qos-ch/logback/blob/v_1.5.7/logback-core/src/main/java/ch/qos/logback/core/util/Duration.java#L96

The unit `second` must be used instead:

```suggestion
      <checkIncrement>5 seconds</checkIncrement>
```